scripts/gulp/test.js

```diff
@@ -33,8 +33,32 @@ function unitTest( cb ) {
   } );
 }
 
+
+/**
+ * Run Mocha JavaScript build tests.
+ * @param {Function} cb - Callback function to call on completion.
+ */
+function buildTest( cb ) {
+  gulp.src( ['./scripts/npm/prepublish/lib/**/*.js'] )
```

Comment:
gulp 4 will likely need this to be `return gulp.src…` so that the function returns a gulp stream.

Reply:
We can wait and see if it complains though.